app/test_main.py

```diff
@@ -1,3 +1,48 @@
+import pytest
+from typing import Any
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
```

Comment:
No need to use class here

Reply:
Using classes allows to logically group related test cases together.

Reply:
We only have tests for get_human_age, so I thought there's no reason to add class
But let it be
Overall good job